### DISCUSSION TITLE
Avalonia.Browser  11.1.0-beta2  test error

### DISCUSSION THREAD

**Discussion by YSW33** [AvaloniaUI/Avalonia#16332] [Q&A]
### Is your feature request related to a problem? Please describe.

Avalonia 11.1.0-beta2
Semi.Avalonia 11.1.0-beta1.1
Irihi.Ursa 0.3.0-beta20240329

![image](https://github.com/user-attachments/assets/4f72a6e2-b3f9-4e8d-afb2-a3e1f822943b)
![image](https://github.com/user-attachments/assets/d4b1dba2-7f7b-40fb-9ecf-7a157e1acd86)
![image](https://github.com/user-attachments/assets/cecfe8d0-d32f-44d4-9e0c-da553f70935b)

### Describe the solution you'd like

![image](https://github.com/user-attachments/assets/82ec4295-fa11-4fd8-a4cc-9c5a42ab5e5e)

**Comment by maxkatz6** [comment]
Can't really help with this information. 
Try to recreate browser project from the templates using `-v 11.1-beta1`  key
